ci/infra/testrunner/tests/test_dockercaps.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import pytest
+import re
+import time
+
+logger = logging.getLogger("testrunner")
+
+
+class NoRootUserError(RuntimeError):
+    def __init__(self, container, output):
+        if 'su' in output:
+            msg = "Container {}: {} -- Regression (see bsc#1142552).".format(container, output)
+        else:
+            msg = "No root user in container {}: {}".format(container, output)
+        super().__init__(msg)
+
+class CannotAddUser(RuntimeError):
+    def __init__(self, container, output):
+        msg = "Cannot add a new user in the container {}: {}".format(container, output)
+        super().__init__(msg)
+
+
+def test_dockercaps(deployment, kubectl):
+
+    logger.info("Deploy testcases")
+    kubectl.run_kubectl(
+        "apply -f https://pastebin.com/raw/iNA7Kh9L")
```

Comment:
I will prefer to have these manifests as part of the tests and not as an external resource in pastebin. You could either embed them as a string in this test or load as a file.

Reply:
so can I do something like this?

```
test_yaml = "blah blah blah"
kubectl.run_kubectl("apply -f", test_yaml)
```